subscription-engine/kafka-ws/app/WebsocketTarball.js

```diff
@@ -118,6 +118,10 @@ class Tarball {
     // TODO: create a class to handle info about the connection and
     // set things like expirate, kafkatopic, etc
 
+    if (!this.kafkaTopicsCallbacksMgmt.getKafkaStatus()) {
```

Comment:
Wouldn't be better to check if the service is healthy?

Reply:
the result of this function returns the healthiness of the service.